File: modules/x-engine-native-cache/android/native-cache/src/main/java/com/zkty/nativ/cache/Nativecache.java
package com.zkty.nativ.cache;

import android.util.Log;

import com.tencent.mmkv.MMKV;
import com.zkty.nativ.core.NativeModule;
import com.zkty.nativ.core.XEngineApplication;

import java.io.File;

public class Nativecache extends NativeModule implements Icache {

    @Override
    public String moduleId() {
        return "com.zkty.native.cache";
    }

    @Override
    public void afterAllNativeModuleInited() {

    }
}

File: modules/x-engine-native-cache/iOS/Class/Native_cache.m
//
//  Native_cache.m
//  cache
//


#import "Native_cache.h"
#import "XENativeContext.h"

@interface Native_cache()
{ }
@end

@implementation Native_cache
NATIVE_MODULE(Native_cache)

 - (NSString*) moduleId{
    return @"com.zkty.native.cache";
}

- (int) order{
    return 0;
}

- (void)afterAllNativeModuleInited{
} 
-(NSString*) test{
    return @"test";
}
@end
 
